desc/grid.py

```diff
@@ -396,20 +396,22 @@ class Grid(_Grid):
     Parameters
     ----------
     nodes : ndarray of float, size(num_nodes,3)
-        node coordinates, in (rho,theta,zeta)
+        Node coordinates, in (rho,theta,zeta)
+    NFP : int
+        Number of field periods (Default = 1).
     sort : bool
-        whether to sort the nodes for use with FFT method.
+        Whether to sort the nodes for use with FFT method.
     jitable : bool
         Whether to skip certain checks and conditionals that don't work under jit.
         Allows grid to be created on the fly with custom nodes, but weights, symmetry
         etc may be wrong if grid contains duplicate nodes.
     """
 
-    def __init__(self, nodes, sort=True, jitable=False):
+    def __init__(self, nodes, NFP=1, sort=True, jitable=False):
```

Comment:
Can you put nfp after sort? Will hopefully minimize any needed changes in existing calls to Grid